simul/platform/deterlab.go

```diff
@@ -91,12 +98,16 @@ var simulConfig *onet.SimulationConfig
 func (d *Deterlab) Configure(pc *Config) {
 	// Directory setup - would also be possible in /tmp
 	pwd, _ := os.Getwd()
-	d.cothorityDir = pwd + "/cothority"
-	d.deterDir = pwd + "/platform/deterlab"
-	d.deployDir = d.deterDir + "/remote"
-	d.buildDir = d.deterDir + "/build"
+	d.simulDir = pwd
+	d.deployDir = pwd + "/deploy"
+	d.buildDir = pwd + "/build"
+	_, file, _, _ := runtime.Caller(0)
```

Comment:
just to get it right, that means deleterlab will set the plartform path to the `main.go` file provided by the user in their own repo, is that right ?